CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+# Contributing to Package Analysis
+
+Hello new contributor! Thank you for contributing your time and expertise to the Package Analysis project.
+We're delighted to have you on board.
+
+This document describes the contribution guidelines for the project.
+
+## Ways to get in touch
+
+If you have any contribution-related questions, please get in touch! Here are some ways to reach current contributors
+1. Open a new issue (strongly preferred)
+1. Via the [OpenSSF Securing Critical Projects Working Group](https://github.com/ossf/wg-securing-critical-projects) mailing list or Slack channel
+
+Note: for minor changes (typos, documentation improvements), feel free to open a pull request directly.
+
+**Note:** Before you start contributing, you must read and abide by our
+**[Code of Conduct](./CODE_OF_CONDUCT.md)**.
+
+## Contributing code
+
+### Getting started
+
+1.  Create [a GitHub account](https://github.com/join)
+1.  Set up your [development environment](#environment-setup)
+
+## Environment Setup
+
+You must install these tools:
+
+1.  [`git`](https://help.github.com/articles/set-up-git/): For source control.
+1.  [`go`](https://go.dev/dl/): For running code.
+1.  `libpcap-dev`: Network packet capture library
```

Comment:
I don't think this is required anymore (although it might come back later)

Reply:
Need to add docker-compose to dependencies for running locally